sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/jdbc/JDBCTable.scala

```diff
@@ -40,6 +40,8 @@ case class JDBCTable(ident: Identifier, schema: StructType, jdbcOptions: JDBCOpt
     util.EnumSet.of(BATCH_READ, V1_BATCH_WRITE, TRUNCATE)
   }
 
+  override def properties(): util.Map[String, String] = jdbcOptions.parameters.toMap.asJava
```

Comment:
we need to define what are the table properties of a JDBC table. Ideally it should only be the properties users put in  `CREATE TABLE ... TBLPROPERTIES`

Reply:
Oh, so we need to support all dialect for different jdbc driver.